libs/ledger-live-common/src/families/tron/bridge/js.ts

```diff
@@ -649,7 +649,7 @@ const getTransactionStatus = async (a: TronAccount, t: Transaction): Promise<Tra
     const { bandwidth, energy } = a.tronResources.frozen;
     if (resource === "BANDWIDTH" && t.amount.gt(bandwidth?.amount || new BigNumber(0))) {
       errors.resource = new TronNoFrozenForBandwidth();
-    } else if (t.amount.gt(energy?.amount || new BigNumber(0))) {
+    } else if (resource === "ENERGY" && t.amount.gt(energy?.amount || new BigNumber(0))) {
```

Comment:
Do you think you could cover this with a case in https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/tron/bridge.integration.test.ts ?

Reply:
@haammar-ledger just added the test. Let me know if it's sufficient. 🙂